cost-analyzer/templates/cost-analyzer-deployment-template.yaml

```diff
@@ -463,6 +463,7 @@ spec:
                   key: prometheus-server-endpoint
             - name: CLOUD_PROVIDER_API_KEY
               value: "AIzaSyDXQPG_MHUEy9neR7stolq6l0ujXmjJlvk" # The GCP Pricing API requires a key.
+                                                               # This GCP api key is expected to be here and is limited to accessing google's billing API. 
```

Comment:
I feel we may be able to communicate this better, i.e. this is a public key for trials. Thoughts on including on a single comment line succinctly? 

@AjayTripathy for input who is most familiar with this key